src/test/groovy/hudson/plugins/gradle/injection/BuildScanInjectionGradleIntegrationTest.groovy

```diff
@@ -216,6 +214,55 @@ class BuildScanInjectionGradleIntegrationTest extends AbstractIntegrationTest {
         gradleVersion << GRADLE_VERSIONS
     }
 
+    def 'Injection is enabled and disabled based on node labels'(String gradleVersion) {
```

Comment:
I think that other tests start with the lower case latters

```suggestion
    def 'injection is enabled and disabled based on node labels'(String gradleVersion) {
```

Reply:
Done